### DISCUSSION TITLE
Gamecube HW2 internal Reset

### DISCUSSION THREAD

**Discussion by Bakuku** [darthcloud/BlueRetro#633] [Q&A]
Hello everybody,

I’m installing a Blueretro mod inside a Gamecube following Nostalgic Indulgence’s Guide but I want to add the reset function described in the Wiki.

If I understand correctly I just have to sever the reset trace (button side), connect it to IO0 and then reestablish the connection from IO14 to the trace (console side), leaving the ESP32 in between, correct?

Any help and advice is welcome.

thank you

**Comment by Colrut** [comment]
just did this on a gamecube.  this assumes using a developer board rather than bare chip.
sever the top corner pin between the switch to the controller pcb.
run a wire from this pin (on the switch) to the esp32 side of the 'BOOT' button

pressing the gamecube reset switch will activate the 'BOOT' button on the ESP32.
this will remove the native function of the reset switch but will allow you to enter the various pairing/reset modes of the ESP32.

![PXL_20230801_144922749](https://github.com/darthcloud/BlueRetro/assets/83892045/36f8364c-4e04-47aa-ac17-4a2e6399315b)
![PXL_20230801_145208500](https://github.com/darthcloud/BlueRetro/assets/83892045/e6cec914-cd5b-45ca-8941-25ff7c95cd5e)
![PXL_20230801_153026499](https://github.com/darthcloud/BlueRetro/assets/83892045/e6323755-0112-4eb4-9722-b06d9bb64400)
